### PR TITLE
Traffic Mirror cleanup 

### PR DESCRIPTION
**Pull request by GregHanson**
automtls wording and remove broken sudo command

[ ] Configuration Infrastructure
[ X ] Docs
[ ] Installation
[ X ] Networking
[ ] Performance and Scalability
[ ] Policies and Telemetry
[ ] Security
[ ] Test and Release
[ ] User Experience
[ ] Developer Infrastructure

### CODE REVIEW

**Comment by ericvn** [comment]
Changing title and adding a label for 1.5.

File: content/en/docs/tasks/traffic-management/mirroring/index.md [hunk 1]
```diff
@@ -137,10 +137,11 @@ In this step, you will change that behavior so that all traffic goes to `v1`.
 
 1.  Create a default route rule to route all traffic to `v1` of the service:
 
-    {{< warning >}}
-    If you installed/configured Istio with mutual TLS authentication enabled, you must add a TLS traffic policy `mode: ISTIO_MUTUAL` to the `DestinationRule` before applying it.
-    Otherwise requests will [generate 503 errors](/docs/ops/common-problems/network-issues/#503-errors-after-setting-destination-rule).
-    {{< /warning >}}
+    {{< tip >}}
```
Comment (frankbu) [inline content/en/docs/tasks/traffic-management/mirroring/index.md]: I think this tip will confuse readers, esp ones that don't understand the history. I suggest just removing the warning and saying nothing else.

Comment (GregHanson) [reply content/en/docs/tasks/traffic-management/mirroring/index.md]: works for me, removed

**Pull request merged**